src/ACEtk/interpretation/ContinuousEnergyNeutron/Builder/EnergyDistribution/Builder/test/Builder.test.cpp

```diff
@@ -0,0 +1,518 @@
+#define CATCH_CONFIG_MAIN
+
+#include "catch.hpp"
+#include "ACEtk.hpp"
+
+using namespace njoy::ACEtk;
+using namespace njoy::ACEtk::interpretation;
+
+
+template< typename B >
+void LAW1( B& ED ){
+  std::vector< int > boundaries{ 0, 3 };
```

Comment:
A boundary value of zero makes no sense here as the index is one based. Does this lead to an error in the test?

Reply:
Very nice catch. I do require the boundaries to be "strictly positive". This unveiled a bug in my [StrictlyPositive](https://github.com/njoy/ACEtk/blob/feature/CENBuilder/src/ACEtk/details/verify/StrictlyPositive/src/check.hpp#L22) check, which has been fixed.

Reply:
Done. Caused 19 tests to fail, but we made the updates.